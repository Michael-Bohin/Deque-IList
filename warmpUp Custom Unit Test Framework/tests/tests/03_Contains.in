new string
add uEDohzsw
add sGoSJQCd
add xpqcmBAh
add mRbuvRco
add YXuekNRh
add FBsIOIhj
add blCsqzAk
add hypGVNJa
add DkTROFMh
add bJpltTIB
count
foreach
count
contains hypGVNJa
contains xpqcmBAh
contains DkTROFMh
contains ahoj
contains svabv
contains 1234567
clear 
foreach
count
contains hypGVNJa
contains xpqcmBAh
contains DkTROFMh
contains ahoj
contains svabv
contains 1234567
add svabv
add 1234567
contains ahoj
contains svabv
contains 1234567

new int
add 17097651
add 37587447
add 99690365
add 37316976
add 17726131
add 76247289
add 61448270
add 30095731
add 71872119
add 15910798
count
foreach
contains 123456
contains 45875
contains 546123654
contains 30095731
contains 15910798
contains 17097651
clear
count
contains 123456
contains 45875
contains 546123654
contains 30095731
contains 15910798
contains 17097651
add 45875
count
contains 123456
contains 45875
contains 546123654
contains 30095731
contains 15910798
contains 17097651
