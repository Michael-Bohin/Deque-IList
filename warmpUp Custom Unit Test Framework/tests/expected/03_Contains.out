10
uEDohzsw
sGoSJQCd
xpqcmBAh
mRbuvRco
YXuekNRh
FBsIOIhj
blCsqzAk
hypGVNJa
DkTROFMh
bJpltTIB
10
true
true
true
false
false
false
0
false
false
false
false
false
false
false
true
true
10
17097651
37587447
99690365
37316976
17726131
76247289
61448270
30095731
71872119
15910798
false
false
false
true
true
true
0
false
false
false
false
false
false
1
false
true
false
false
false
false
